onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -1509,10 +1509,22 @@ def aten_embedding_sparse_backward(
     raise NotImplementedError()
 
 
-def aten_empty_like(self: TensorType, memory_format: Optional[str] = None) -> TensorType:
+@torch_op("aten::empty")
+def aten_empty(size, dtype: int = 1):
+    # empty(SymInt[] size, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None, MemoryFormat? memory_format=None) -> Tensor
+
+    # using RandomUniform value to simulate np.empty()
+    result = op.RandomUniform(dtype, 1e10, -1e10, 0.5, size)
```

Comment:
Is RandomUniform needed, or can it just be a bunch of zeros?

Reply:
Since the function just need to produce _any_ tensor

Reply:
I agree. zeros may be better than random. Even that looks a bit unsatisfactory, since presumably the returned tensor is subsequently updated imperatively to fill in the values. A better exporter might be able to eliminate the use of such ops with a better analysis of the context where the result of this op is used. But as a fallback, a tensor with zeros may be ok.